File: py/problem_1.py
# Problem 1
def sumOfMultiplesOf3Or5(n):
	sum1 = 0

	for i in range(n):
		if i % 3 == 0 or i % 5 == 0:
			sum1 += i
	return sum1

if __name__ == "__main__":
	sum1 = sumOfMultiplesOf3Or5(1000)
	print("The Sum of Multiples of 3 or 5 below 1000 is {}".format(sum1))

File: julia/problem_1.jl
N = 1000
sum = 0

for i in 0:(N-1)
	if (i % 3 == 0) || (i % 5 == 0)
		global sum += i
	end
end

println("Sum of multiples of 3 and 5 under 1000 is: ", sum)

